### PR TITLE
[ENH] docker workflow for pyafq and afqsi

### PR DESCRIPTION
**Pull request by 36000**
Should automate the creation of a pyAFQ python version 0.8 image and pyafq/qsi python version 0.7 image

### CODE REVIEW

**Comment by 36000** [comment]
This closes #62 .

File: .github/workflows/docker_afqsi.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
```
Comment (arokem) [inline .github/workflows/docker_afqsi.yml:32]:
```suggestion
    - name: Login to GHCR
```

File: .github/workflows/docker_pyafq.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
```
Comment (arokem) [inline .github/workflows/docker_pyafq.yml:15]:
```suggestion
    - name: Login to GHCR
```

File: afqsi_docker/docker-build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+COMMIT=${1}
+COMMIT="$(echo "${COMMIT}" | tr -d '[:space:]')"
+export COMMIT
+TAG="ghcr.io/${2}/afqsi:${COMMIT}"
+TAG="$(echo "${TAG}" | tr -d '[:space:]')"
+echo $TAG
+docker build --no-cache -t $TAG --build-arg COMMIT ./afqsi_docker
+docker run --rm -i --entrypoint /usr/local/miniconda/bin/pytest $TAG /usr/local/miniconda/lib/python3.7/site-packages/AFQ/tests/test_api.py::test_AFQ_data_waypoint
```
Comment (arokem) [inline afqsi_docker/docker-build.sh:8]: Does this mean that the test is run every time a user calls this script to build the docker?

Comment (arokem) [reply afqsi_docker/docker-build.sh:8]: Or do we never use these scripts to build? The builds are only ever done on GitHub Actions?

**Review by arokem — COMMENTED** [review]
Would be great to add some documentation about this, but we can also do that on a separate PR.

File: afqsi_docker/docker-build.sh (hunk 3 above)
Comment (36000) [reply afqsi_docker/docker-build.sh:8]: The latter. The user is not meant to use this script. We use this script to build. I want to test that pyAFQ installed correctly in the qsiprep environment everytime we make a new image. I could probably have found a way to write all of this inside the workflow file, but I am more familiar with bash syntax, so I wrote these two little shell scripts instead.

**Pull request merged**